src/metadata_tool/sources/metadata.clj

```diff
@@ -303,6 +304,33 @@
   "The metadata for a specific activity, identified by name."
   (memoize activity-metadata-by-name-fn))
 
+(defn- includes?
```

Comment:
i believe the idiom is to use `(some? (set element) list)` and not even use this helper function.

Reply:
I managed to implement it with `(some #(= lower-repo-name %) (set lower-case-repos))` ; the `some?` function only takes one `list` parameter.